hik_camera:
  ros__parameters:
    camera_sn: "00E89503815" 
    
    camera_info_url: ""
    
    exposure_time: 10000.0
    gain: 10.0
    frame_rate: 60.0